core/src/main/scala/scalaz/syntax/EitherTOps.scala

```diff
@@ -0,0 +1,15 @@
+package scalaz
+package syntax
+
+import Leibniz.===
+
+final class EitherTOps[V](val self: V) extends AnyVal {
```

Comment:
```diff
- final class EitherTOps[V](val self: V) extends AnyVal {
+ final class EitherTOps[V](private val self: V) extends AnyVal {
```

Reply:
what's the advantage of adding `private` ?